README.md

```diff
@@ -11,16 +11,27 @@
 [![Rating](https://img.shields.io/visual-studio-marketplace/r/WatermelonTools.watermelon-tools?style=flat-square)](https://marketplace.visualstudio.com/items?itemName=WatermelonTools.watermelon-tools&ssr=false#review-details)
 [![Slack](https://img.shields.io/badge/Slack%20Community-Watermelon-brightgreen)](https://join.slack.com/t/watermelonusers/shared_invite/zt-15bjnr3rm-uoz8QMb1HMVB4Qywvq94~Q)
 
-Watermelon is an **open-source** integration between **GitHub** and **Visual Studio Code** to document code and view Git Blame. Watermelon makes you an expert on any file instantly by running `git blame` for you and telling you why a block of code was written that way by someone else.
+Watermelon is an **open-source** integration between **GitHub** and **Visual Studio Code** to help developers go beyond Git Blame. Watermelon makes you an expert on any file instantly by running `git blame` for you and telling you why a block of code was written that way by someone else.
 
 ## Features
 
-Watermelon has 2 actions that you can use:
+Watermelon has 3 main features that you can use:
 
 - View Git Blame
 - View Pull Requests
+- Daily Summary
 
-Simply click the Watermelon icon on the sidebar. After that you can click any of the 2 different buttons Watermelon has.
+The main UI to use Watermelon is the context box you see when you hover over lines of code with our extension installed. This UI is complemented with the sidebar we have.
+
+In the image below you can see how hovering over a line of code provides us the following data:
+
+![hover](https://i.ibb.co/rdvgVnS/hover-showcase.gif)
```

Comment:
Shouldn't these media files live in [`/images`](https://github.com/watermelontools/wm-extension/tree/dev/media)?